packages/next/shared/lib/loadable.js

```diff
@@ -144,6 +151,12 @@ function createLoadableComponent(loadFn, options) {
     }, [props, state])
   }
 
+  function LazyImpl(props, ref) {
+    useDynamicModules(opts.modules)
```

Comment:
Does this mean that the client isn't going to hydrate until the modules are loaded? I'm not sure that's the behavior we want, as we don't benefit from anything like selective hydration that way.

Reply:
Removed. That was added but doesn't really take effects.